test/opengl/user_state_check.cpp

```diff
@@ -0,0 +1,82 @@
+#include <csetjmp>
+#include <unistd.h>
+
+#if defined(__APPLE__)
```

Comment:
@abadams I'm confused, `user_state_check.cpp` and `rewrap_texture.cpp` both have the same boilerplate for including GL headers:

```cpp
#if defined(__APPLE__)
#include <OpenGL/gl.h>
#else
#include <GL/gl.h>
#endif
```

Is there something else I'm not seeing?